### PR TITLE
fix build and run for 64-bit Linux, need to search dep/lib64 too

### PR DESCRIPTION
**Pull request by av500**
Signed-off-by: Vladimir Pantelic <vladimir@pantelic.net>

### CODE REVIEW

**Comment by av500** [comment]
Hi Andrew, 

this is what I still have on my side, feel free to drop what your don't like but at least the lib64 issue should be handled.

File: Makefile [hunk 1]
```diff
@@ -44,7 +44,7 @@ all: $(TARGET)
 
 run: $(TARGET)
 ifeq ($(ARCH), lin)
-	LD_LIBRARY_PATH=dep/lib ./$<
+	LD_LIBRARY_PATH=dep/lib:dep/lib64 ./$<
```
Comment (AndrewBelt) [inline Makefile]: If libraries are being put in lib64, we need to fix that instead. Everything should go in dep/lib

Comment (av500) [reply Makefile]: ok, let me see if I can get that done

**Comment by AndrewBelt** [comment]
Closing because obsolete

**Pull request closed without merging**